src/diffusers/pipelines/stable_diffusion/__init__.py

```diff
@@ -3,4 +3,9 @@
 
 
 if is_transformers_available():
-    from .pipeline_stable_diffusion import StableDiffusionPipeline, StableDiffusionSafetyChecker
+    from .pipeline_stable_diffusion import (
+        StableDiffusionImg2ImgPipeline,
+        StableDiffusionInPaintPipeline,
```

Comment:
I don't think `StableDiffusionImg2ImgPipeline` and `StableDiffusionInPaintPipeline` are visible there.

Reply:
The pipelines are moved into `src/diffusers/pipelines/` and `examples/inference` is outdated - will try to make this clearer in the readme :-)